src/features/courses/course-controller.ts

```diff
@@ -2236,12 +2247,80 @@ class CourseController {
                 END
             `);
 
-            attributes = [
-                [averageScoreAttribute, 'average'],
-                [sequelize.literal(pendingProblemCountCalculationString), 'pendingProblemCount'],
-                [sequelize.literal(masteredProblemCountCalculationString), 'masteredProblemCount'],
-                [sequelize.literal(inProgressProblemCountCalculationString), 'inProgressProblemCount'],
-            ];
+            // If the topicId isn't present (and implicitly, the questionId as well),
```

Comment:
Why are we making this assumption? Why not show the open dead and closed score no matter what the filter is

Honestly, it is probably most important at the lowest filter level, just the course